Documentation/HowToUseWCFInContainers.md

```diff
@@ -0,0 +1,142 @@
+    </PackageReference>
+    <PackageReference Include="System.ServiceModel.Primitives">
+      <Version>4.3.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.ServiceModel.Duplex">
+      <Version>4.3.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.ServiceModel.Security">
+      <Version>4.3.0</Version>
+    </PackageReference>
+  </ItemGroup>
+</Project>
+~~~
+
+#### To sum it up: you have a WCF default template (or your own) service running and a folder called �wcfClient� with the client app that has 3 files: wcfClient.csproj, program.cs, and IService1.cs.
+
+### 2. Container base image
+.NET Core Runtime Docker images are available for both released versions and for nightly builds (https://hub.docker.com/r/microsoft/dotnet-nightly/). One can get images that have only the .NET Core runtime or both the runtime and the SDK. We�re going to use Nano Server and Linux SDK images for the latest 1.1.1 nightly build (which are compatible with the project file that we created above).
+
+### 3. Dockerfile
+Our goal here is to have a docker file that restores necessary packages and builds the app at the time when we build the container. Should you pick up a different base layer and rebuild your container � the app will automatically use the new binaries. Having a container with a shared .NET Core framework helps to save space and to speed up launching new containers with the same base image.
+
+#### Dockerfile for Linux
+~~~
+FROM microsoft/dotnet-nightly:1.1.1-sdk
+COPY wcfClient wcfClient
+RUN dotnet restore wcfClient/wcfClient.csproj
+RUN dotnet build wcfClient/wcfClient.csproj
+CMD dotnet run -p wcfClient/wcfClient.csproj
+~~~
+
+#### Dockerfile for Windows Nano Server
+~~~
+FROM microsoft/dotnet-nightly:1.1.1-sdk-nanoserver
+COPY wcfClient wcfClient
+RUN dotnet restore wcfClient\wcfClient.csproj
+RUN dotnet build wcfClient\wcfClient.csproj
+CMD dotnet run -p wcfClient\wcfClient.csproj 
+~~~
+### 4. Build and run the container
+Navigate to the folder that contains the dockerfile and a subfolder called �demo� and run the following commands. Since our sample app reads an environment variable named "url" to create an endpoint we pass it via --env parameter:
+
+~~~
+docker build -t wcf_client_in_container .
+docker run --env url=http://server_name/app_name/Service1.svc -it --rm wcf_client_in_container
+~~~
+
+### 5. Certificates 
+Windows Nano Server has powershell cmdlets to install client certificates. If your certificate authority public key is in cert_authority_public.cer  and client certificate is client_cert.pfx (protected by pwd �password�) then the dockerfile will have the following powershell commands:
+~~~
+$pass = ConvertTo-SecureString -AsPlainText -Force password 
+Import-pfxcertificate -certStoreLocation Cert:LocalMachine\My -Password $pass -FilePath client_cert.pfx
+Import-Certificate -certStoreLocation Cert:LocalMachine\Root -FilePath cert_authority_public.cer
+~~~
```

Comment:
So this works only in windows container, right? How about Linux? Where should these cert files live? How did files shared between the host and the container?

It's a little bit weird to see this section as there is no previous scenario using certificate. How about you update the sample code in this section to show how certs are used?